src/bluehawk/tags/removeStateTag.test.ts

```diff
@@ -0,0 +1,94 @@
+import { fail } from "yargs";
+import { Bluehawk } from "../bluehawk";
+import { Document } from "../Document";
+import { RemoveStateTag } from "./RemoveStateTag";
+import { StateTag } from "./StateTag";
+
+describe("remove state tag", () => {
+  const bluehawk = new Bluehawk();
+  bluehawk.registerTag(RemoveStateTag);
+  bluehawk.registerTag(StateTag);
+  bluehawk.addLanguage("js", {
+    languageId: "javascript",
+    blockComments: [[/\/\*/, /\*\//]],
+    lineComments: [/\/\/ ?/],
+  });
+
+  it("strips text", async (done) => {
```

Comment:
This should work without accepting/calling `done` just by nature of it being async... try it and see?

Reply:
You are correct. Looked at Jest docs and it seems that this is only necessary when you need to indicate completion of a test from a callback, thanks for pointing this out :)